app/assets/stylesheets/govuk_publishing_components/application.scss

```diff
@@ -46,6 +46,23 @@ $border-color: #ccc;
   }
 }
 
+.component-violation {
+  border: 3px solid $error-colour;
+  margin: 0 0 $gutter;
+  padding: $gutter $gutter;
+
+  .component-violation--title {
```

Comment:
I think this should be using underscores instead of double dash, e.g: component-violation__title. -- is used for [modifiers](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#bem)

Reply:
Fixed